.github/workflows/reusable-zizmor.yml

```diff
@@ -348,8 +348,13 @@ jobs:
             echo "always-use-default-config is set. Using default config."
           fi
 
+          # Some forks are having issues with cloning the zizmor config file, so in the worst case
+          # we'll use the default configuration that's built in.
+
```

Comment:
We've already covered this extensively in the message above (line 107). _All_ forks will fail and fall back to the default. I'd drop this - we don't need to say again.

```suggestion
```